projects/storefrontlib/src/cms-pages/product-details-page/product-details-page.module.ts

```diff
@@ -1,8 +1,9 @@
 import { NgModule } from '@angular/core';
 import { RouterModule } from '@angular/router';
+import { ConfigModule, RoutingConfig } from '@spartacus/core';
 import { CmsPageGuard } from '../../cms-structure/guards/cms-page.guard';
 import { PageLayoutComponent } from '../../cms-structure/page/page-layout/page-layout.component';
-import { suffixUrlMatcher } from '../../cms-structure/routing/suffix-routes/suffix-url-matcher';
+import { PRODUCT_DETAILS_URL_MATCHER_FACTORY } from './product-details-url-matcher-factory';
```

Comment:
`PRODUCT_URL_MATCHER` ?
`PRODUCT_ROUTE_MATCHER`?

Next to:
`CATEGORY_ROUTE_MATCHER` and `SEARCH_ROUTE_MATCHER`?

Or  `PDP_ROUTE_MATCHER` vs `PLP_ROUTE_MATCHER`.... don't like that too much actually...

Reply:
Or:
`PRODUCT_URL_MATCHER`
`PRODUCT_LISTING_URL_MATCHER`